### PR TITLE
WIP ;;prune rewrite and ;;skipto command

### PR DESCRIPTION
**Pull request by knight-ryu12**
This is **WIP** PR. I/we add many commit afterward.

So this is some core idea.

### CODE REVIEW

File: FredBoat/config.yaml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 ---
 patron:            false       # Set this to true for self hosting the music bot
 development:       true        # Set this to true for self hosting the full bot (including non music commands)
-prefix:            '<<'        # Default prefix used by the bot
+prefix:            '!?'        # Default prefix used by the bot
 restServerEnabled: true        # Set this to false if you are running multiple FredBoat bots on the same machine
-admins:            []          # Add comma separated userIds and roleIds that should have access to bot admin commands
+admins:            ["135788476841852929"]          # Add comma separated userIds and roleIds that should have access to bot admin commands
```
Comment (Frederikam) [inline FredBoat/config.yaml]: Please revert changes to this file

Comment (knight-ryu12) [reply FredBoat/config.yaml]: oh yeah.
I'll do that.

File: FredBoat/src/main/java/fredboat/command/music/control/SkipToCommand.java [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class SkipToCommand extends Command implements IMusicCommand, ICommandRestricted {
+
+    // Proposal wait. do not work!
+    // Napster will change GuildPlayer.
```
Comment (Frederikam) [inline FredBoat/src/main/java/fredboat/command/music/control/SkipToCommand.java]: I'm confused here. Are you requesting this to be pulled or not?

**Comment by knight-ryu12** [comment]
also why other PR is not giving me Cl error

**Comment by Frederikam** [comment]
Closes #49

**Comment by Shredder121** [comment]
Pretty sure you need to put that in the pull request body.

**Comment by napstr** [comment]
> also why other PR is not giving them Cl error

Your PR is not compliant with at least one of the tests. You can check the travis log to find the details. One of your commands does not provide a proper answer in the help() method.

**Comment by knight-ryu12** [comment]
Since I need to rebase a lot of settings, So I gonna recreate entire thing.

**Pull request closed without merging**